src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < 3; i++) {
+            System.out.println(figureSupplier.getRandomFigure().infoFromRigure());
```

Comment:
The method call `infoFromRigure` seems to be a typo. It should be `infoFromFigure` to match the corrected method name in the `Figure` class and its subclasses.